docs/book/src/tasks/installation.md

```diff
@@ -95,6 +158,15 @@ curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"in
   | kubectl create -f -
 ```
 
+{{#/tab }}
+{{#tab Docker}}
+
+Check the [Docker provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-docker/releases) for an up-to-date components file.
+
+```bash
+kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-docker" asset:"provider_components.yaml" version:"0.2.x"}}
```

Comment:
Should we leave this step as a TODO for now, since this will not be the way that it would be deployed for v0.3.0 or any pre-releases we do for v0.3.x?

Reply:
I can open an issue about this, given that we don't have a release pipeline or artifacts for CAPD, this still stands for now, wdyt?

Reply:
sgtm

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1565